.github/workflows/build.yml

```diff
@@ -51,6 +51,80 @@ jobs:
           path: compiler/cpp/thrift
           retention-days: 3
 
+  lib-go:
+    needs: compiler
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.17.0'
```

Comment:
the current supported go versions are 1.18.x and 1.19.x: https://github.com/apache/thrift/blob/master/LANGUAGES.md, not 1.17.x.

but looks like it actually picks the latest version (1.19.1) from the logs, so I think we probably actually need two actions, one for `lib-go-1.18` with `<1.19.0` and one for `lib-go-1.19` with `>=1.19.0`, or something like that.

Reply:
thanks for the suggestion - will setup a new jira to follow up

Reply:
see https://issues.apache.org/jira/browse/THRIFT-5651